.clang-format

```diff
@@ -5,7 +5,7 @@ TabWidth: 2
 IndentWidth: 2
 IncludeBlocks: Preserve
 NamespaceIndentation: None
-PointerAlignment: Middle
+PointerAlignment: Left
```

Comment:
I don't find this acceptable: all OS kernels we're going to work with have this to the right: Linux Kernel, ThreadX, FreeRTOS, uC/OS, etc.
Creating a different style would cause confusion between the IoT SDK core and the ports.

As discussed offline, Middle is a good compromise (although deep inside, I know right is right 😜 )

Reply:
@danewalton, so Middle or Left? :)

Reply:
Cris and I talked offline and believe we can go left (maintains consistent style with the iot sdk). I recognize that it won't make everyone happy but I think ultimately middle leaves no one happy. It looks a lot like multiplication and very very few people use it. For the sake of choosing one and moving on to more important things, the majority of the IoT team is good with left.

Reply:
If we can't do right let's at least remove the extra space...

Reply:
We're about to wrap up most of the planned work soon. I'll apply this PR and re-format all the code.

Reply:
Thanks @antkmsft - really appreciate the flexibility, allowing our preference as well as handling the extra work this generated!